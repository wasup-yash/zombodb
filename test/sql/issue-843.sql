CREATE TABLE issue843 (
                                         id SERIAL8 NOT NULL PRIMARY KEY,
                                         name varchar NOT NULL
);
CREATE INDEX es_idxwildcardedkeywordtest
    ON issue843 USING zombodb ((issue843.*));

INSERT INTO issue843 (name) values ('Pancake');
INSERT INTO issue843 (name) values ('Waffle');
INSERT INTO issue843 (name) values ('French Toast');
INSERT INTO issue843 (name) values ('Cinnamon Toast');

SELECT * FROM issue843 order by id;
SELECT * FROM issue843 WHERE issue843 ==> 'name = "*Toast"' order by id;
SELECT * FROM issue843 WHERE issue843 ==> 'name = "*Toa*"' order by id;
SELECT * FROM issue843 WHERE issue843 ==> 'name = "*ench Toa*"' order by id;

DROP TABLE issue843;