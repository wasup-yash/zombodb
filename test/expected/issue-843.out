CREATE TABLE issue843 (
                                         id SERIAL8 NOT NULL PRIMARY KEY,
                                         name varchar NOT NULL
);
CREATE INDEX es_idxwildcardedkeywordtest
    ON issue843 USING zombodb ((issue843.*));
INSERT INTO issue843 (name) values ('Pancake');
INSERT INTO issue843 (name) values ('Waffle');
INSERT INTO issue843 (name) values ('French Toast');
INSERT INTO issue843 (name) values ('Cinnamon Toast');
SELECT * FROM issue843 order by id;
 id |      name      
----+----------------
  1 | Pancake
  2 | Waffle
  3 | French Toast
  4 | Cinnamon Toast
(4 rows)

SELECT * FROM issue843 WHERE issue843 ==> 'name = "*Toast"' order by id;
 id |      name      
----+----------------
  3 | French Toast
  4 | Cinnamon Toast
(2 rows)

SELECT * FROM issue843 WHERE issue843 ==> 'name = "*Toa*"' order by id;
 id |      name      
----+----------------
  3 | French Toast
  4 | Cinnamon Toast
(2 rows)

SELECT * FROM issue843 WHERE issue843 ==> 'name = "*ench Toa*"' order by id;
 id |     name     
----+--------------
  3 | French Toast
(1 row)

DROP TABLE issue843;
